### PR TITLE
ID-43

### PR DESCRIPTION
**Pull request by SharmaDinesh**

### CODE REVIEW

**Comment by bitcoinbrisbane** [comment]
Ill wait for #39 to be merged first

File: app/screens/claim/index.tsx [hunk 1]
```diff
@@ -1,21 +1,80 @@
...
+    try {
+      // Ask the user for the permission to access the camera
+      const permissionResult = await ImagePicker.requestCameraPermissionsAsync();
+      if (permissionResult.granted === false) {
+        alert("You've refused to allow this appp to access your camera!");
+        return;
+      }
+      const result = await ImagePicker.launchCameraAsync();
+      // Explore the result
+      console.log(result);
```
Comment (xiangzhe0812) [inline app/screens/claim/index.tsx:69]: can we remove console.log

**Review by xiangzhe0812 — APPROVED** [review]

File: app/screens/claim/index.tsx [hunk 2]
```diff
@@ -67,6 +67,24 @@ const Claim = () => {
     }
   };
 
+  const uploadPhotoFromCamera = async () => {
+    try {
+      // Ask the user for the permission to access the camera
+      const permissionResult = await ImagePicker.requestCameraPermissionsAsync();
+      if (permissionResult.granted === false) {
+        alert("You've refused to allow this appp to access your camera!");
```
Comment (Mitchell139) [inline app/screens/claim/index.tsx:75]:
```suggestion
        alert("You've refused to allow this app to access your camera!");
```

**Comment by Mitchell139** [comment]
@SharmaDinesh Can you fix the merge conflict

**Review by xiangzhe0812 — APPROVED** [review]

**Pull request closed without merging**